example_test.go

```diff
@@ -223,3 +224,42 @@ func ExampleEnum() {
 	}
 	// Output:
 }
+
+func ExampleCustomString() {
+	a2z := "abcdefghijklmnopqrstuvwxyz"
+	a2z0to9 := "abcdefghijklmnopqrstuvwxyz0123456789"
+
+	// example for generating custom string within one unicode range.
+	var A string
+	unicodeRange := fuzz.UnicodeRange{'a', 'z'}
+
+	f := fuzz.New().Funcs(unicodeRange.CustomStringFuzzFunc())
+	f.Fuzz(&A)
+
+	for i := range A {
+		if !strings.ContainsRune(a2z, rune(A[i])) {
+			fmt.Printf("A[%d]: %v is not in range of a-z.\n", i, A[i])
+		}
+	}
+	fmt.Println("Got a string, each character is selected from  a-z.")
+
+	// example for generating custom string within multiple unicode range.
+	var B string
+	unicodeRanges := fuzz.UnicodeRanges{
+		{'a', 'z'},
+		{0x0030, 0x0039}, // 0 is 0x0030, 9 is 0x0039
```

Comment:
nit: '0' and '9' are more readable...

Reply:
Sorry about for this...

Previously, I considered that `a-z` has been used above, and the following can use `0x0030-0x0039` to instead of  `0-9`. I thought this will let developers know that there are two ways to pass in.

I have changed it according your suggestion, and put `0x0030-0x0039` in the comment.